### PR TITLE
CC | version: Specify forked / vanilla entries of containerd

### PR DESCRIPTION
**Pull request by fidencio**
As we'll need to test with both the vanilla and the forked versions of containerd, let's make sure we'll specify both entries as part of our versions.yaml file, and we can read whatever we need accordingly as part of our tests jobs.

NOTE: This will never make into the `main` branch, but we need this right now in order to have our tests properly setup.

### CODE REVIEW

**Comment by fidencio** [comment]
Please, **DO NOT RUN `/test` HERE**, at least not yet.

File: versions.yaml
Comment (stevenhorsman) [inline versions.yaml:238]: Are you happy for us to be ahead of the `main` versions.yaml here? I'm not against this, but it means that we'd probably need to bump containerd in `main` before we merge this function to `main` if we aren't confident that it works on 1.6.8

Comment (fidencio) [reply versions.yaml:238]: So, we're already testing on `main` with the `latest` containerd (at least the cri-containerd tests).  The k8s tests are relying on what k3s is using, which is also 1.7+

Comment (fidencio) [reply versions.yaml:238]: It's very hard to compare this with `main`, as we have changed the way we consume those for the tests on main.

Comment (stevenhorsman) [reply versions.yaml:238]: Cool - fair point, if AKS is using containerd 1.7+ and we can kill on Jenkins soon then this problem goes away a bit

**Review by stevenhorsman — APPROVED** [review]
LGTM assuming the tests are passing

**Comment by fidencio** [comment]
TEEs issues are not related to this PR.

**Pull request merged**
